contrib/raftexample/kvstore.go

```diff
@@ -77,6 +77,7 @@ func (s *kvstore) readCommits(commitC <-chan *string, errorC <-chan error) {
 			if err := s.recoverFromSnapshot(snapshot.Data); err != nil {
```

Comment:
@xiang90 This is where we recover the initial snapshot. I manually write some keys and confirm that it correctly recovers the initial kv-store from snapshot.